packages/block-editor/src/components/image-size-control/test/index.js

```diff
@@ -272,15 +233,12 @@ describe( 'ImageSizeControl', () => {
 				/>
 			);
 
-			const button = screen.getByRole( 'button', {
+			const button = screen.getByRole( 'radio', {
 				name: '50%',
-				pressed: false,
```

Comment:
Should we add `checked: false` here, instead of the previous `pressed` ?

Reply:
This is now updated.